### PR TITLE
Committers stats based ob ML

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

**Comment by slawekjaranowski** [comment]
example output:
image

image

**Review by slachiewicz — APPROVED** [review]

File: src/main/resources/committers-names.properties [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# additional names used on ML by committers
+
+brianf = Brian Fox
+cstamas = Tamás Cservenák
+hboutemy = Hervé Boutemy
```
Comment (cstamas) [inline src/main/resources/committers-names.properties:22]: isn't this in parent POM?

**Review by cstamas — APPROVED** [review]

File: src/main/resources/committers-names.properties (hunk 1 above)
Comment (slawekjaranowski) [reply src/main/resources/committers-names.properties:22]: I reade names from - https://projects.apache.org//json/foundation/people_name.json
But someone can use a little different name according to national chars and so ...

Comment (olamy) [reply src/main/resources/committers-names.properties:22]: haha US system. be sure dates have correct format :)

**Pull request merged**